python/dask_cudf/dask_cudf/backends.py

```diff
@@ -75,19 +75,39 @@ def _nonempty_index(idx):
     raise TypeError(f"Don't know how to handle index of type {type(idx)}")
 
 
+def _nest(x, n):
+    """
+    Helper for _get_non_empty_data which creates nested data
```

Comment:
Maybe just clarify that this is only for nested list data?

Do we need to handle somthing similar for struct columns?